### ISSUE TITLE
about pgproxy

### ISSUE THREAD

**Issue by lulufeiacount** [goodplayer/pgproxy#1]
Hello author。Is this project proxying the front-end MySQL protocol request, forwarding the request to the back-end postgresql, and then responding with the MySQL protocol? :)

**Comment by goodplayer** [comment]
Yes. The original plan is to translate mysql requests to postgresql requests in order for mysql only applications. The frontend protocol is running MySQL protocol. While the backend runs on postgresql.

However since there should be extremely effort on translation layer, I may not be able to make this happen as soon as possible. So the support of translation will be slowly updated.

**Comment by lulufeiacount** [comment]
Thanks for your reply， I think I have the same functional requirements as you，the query response of MySQL protocol is very complex， looking forward to your project progress。 :)
